pkg/catalog/catalog.go

```diff
@@ -158,7 +222,7 @@ func getDefaultBuilderImages(client *occlient.Client) ([]CatalogImage, error) {
 			// the 'hidden' tag. If so, this builder image is deprecated and should not be offered to the user
 			// as candidate
 			for _, tag := range allTags {
-				imageStreamTag := imageStreamTagMap[imageStream.Name+":"+tag]
+				imageStreamTag := ism[imageStream.Name+":"+tag]
```

Comment:
can we rename `ism` to a better variable name?

Reply:
resolved